### PR TITLE
NC22 compliant

### PR DESCRIPTION
**Pull request by R0Wi**

### CODE REVIEW

**Comment by R0Wi** [comment]
@bahnwaerter ready 4 review. Quick manual test would be appreciated :smile:

File: .vscode/launch.json [hunk 1]
```diff
@@ -9,13 +9,13 @@
             "name": "Listen for XDebug",
             "type": "php",
             "request": "launch",
-            "port": 9000
+            "port": 9003
```
Comment (bahnwaerter) [inline .vscode/launch.json:12]: The new XDebug port configuration does not match the documented port in the project's README. Please do either document the new port value in the project's README or undo this change to preserve the old port value. I would prefer the second possibility to avoid misconfigurations in already existing development setups.

File: .vscode/launch.json [hunk 2]
```diff
@@ -9,13 +9,13 @@
...
         },
         {
             "name": "Listen for XDebug (CLI)",
             "type": "php",
             "request": "launch",
-            "port": 9001
+            "port": 9003
```
Comment (bahnwaerter) [inline .vscode/launch.json:18]: The new XDebug port configuration does not match the documented port in the project's README. Please do either document the new port value in the project's README or undo this change to preserve the old port value. I would prefer the second possibility to avoid misconfigurations in already existing development setups.

**Review by bahnwaerter — COMMENTED** [review]
Thanks for providing a patch to support Nextcloud in version 22. Please look at the changed but undocumented XDebug port configurations for VS Code in your patch and consider my proposed solutions to deal with problems of those port configuration changes.

**Comment by R0Wi** [comment]
@bahnwaerter done some extra modification on our docs. Now ready for review :smiley:

**Review by bahnwaerter — APPROVED** [review]
Thanks @R0Wi for updating the README to document the new XDebug port configurations. So everything seems to be ready for a new release.
